x/fundraising/keeper/auction.go

```diff
@@ -239,9 +238,10 @@ func (k Keeper) CancelAuction(ctx sdk.Context, msg *types.MsgCancelAuction) erro
 	return nil
 }
 
-// AddAllowedBidders is a function for an external module and it simply adds new bidder(s) to AllowedBidder list.
-// Note that it doesn't do auctioneer verification because the module is generalized for broader use cases.
-// It is designed to delegate to an external module to add necessary verification and logics depending on their use case.
+// AddAllowedBidders is a function that is implemented for an external module to add
+// allowed bidders in the auction's AllowedBidders list; it doesn't do the auctioneer's verification
+// because the module is fundamentally designed to delegate authorization to an external module.
+// It is suggested for an external module to add any necessary verification and operations depending on their use cases.
```

Comment:
Does this mean that, the others (not the auctioneer) can call this function from outside of this module? So, is how to restrict the usage of this function up to the external module?

Reply:
The module is designed to delegate authorization to an external module to adopt broader use cases. It means that an external module chooses/decides who can be an auctioneer to create an auction. The function is not exposed in message level so that there is no way to access it unless an external module imports the fundraising module. If this is not clear enough, I can explain to you about this offline.